bootstrap-app/src/cmr/bootstrap/services/bootstrap_service.clj

```diff
@@ -125,55 +126,85 @@
     (info "Waiting" sleep-secs "seconds so indexer index set hashes will timeout.")
     (Thread/sleep (* 1000 sleep-secs))))
 
+(defn- validate-target
+  "Validates the target index is set to a valid value."
+  [target]
+  (when-not (some #{target} [:separate-index :small-collections nil])
```

Comment:
I gave this a +1, but I want to also say this is 1) a good catch, and 2) a _really_ great idea. Thanks @daniel-zamora.

Reply:
It was on my list to do and missed. I was also going to refactor out the similar logic in bootstrap and index-set into a location in the common-app library instead of duplicating. I'll do that now.

Reply:
Done.